index.js

```diff
@@ -1,17 +1,90 @@
-var menu = document.getElementById('menu-items');
-var nav = document.getElementById('navBar')
-var navX = document.getElementById('navX')
+const menu = document.getElementById('menu-items');
+const nav = document.getElementById('navBar');
+const navX = document.getElementById('navX');
+const detail = document.getElementById('portfolio-details');
+const detailImg = document.getElementById('detailImg');
+const detailH1 = document.getElementById('detailH1');
+const detailUl = document.getElementById('detailUl');
+const detailP = document.getElementById('detailP');
+const Details = [
+  {
+    img: 'images/details-image.png',
+    h1: 'Keeping track of hundreds of components',
+    ul: ['Ruby on rails', 'css', 'JavaScript'],
+    ul2: ['Codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Terminal'],
+    p: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+  {
+    img: 'images/details-image1.png',
+    h1: 'Keeping track of hundreds of components',
+    ul: ['Ruby on rails', 'css', 'JavaScript'],
+    ul2: ['Codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Terminal'],
+    p: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
```

Comment:
- [ ] Dear @MarsIncarnate, I see you have an array of objects and that is good, But Its repetitive content and is not according to the project detail. Each object should contain information that could be used to render a dynamic project that has `name`,  `description`, `image`, `technologies`, `link`
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
> Thanks to these changes adding projects in the future will be easy. You will only add it in one place.